### PR TITLE
MicroOS: Change to user prompt instead of using "runuser"

### PR DESCRIPTION
**Pull request by jlausuch**
runuser is not a valid way to switch to a user cause it doesn't
provide a valid systemd session.

https://bugzilla.opensuse.org/show_bug.cgi?id=1184266

- Related ticket: https://progress.opensuse.org/issues/90605

### CODE REVIEW

**Review by sysrich — APPROVED** [review]

**Comment by jlausuch** [comment]
This doesn't really solve the problem. Working now on changing user by login prompt.

**Comment by jlausuch** [comment]
VR (in progress): https://openqa.opensuse.org/tests/1692703

File: tests/microos/toolbox.pm [hunk 1]
```diff
@@ -53,17 +64,19 @@ sub run {
...
+    die "$user shouldn't have access to /etc/passwd!" if (script_run('toolbox -u touch /etc/passwd') == 0);
+
+    record_info 'Test',                               "Rootfull toolbox as $user";
+    validate_script_output 'toolbox -r id',           sub { m/uid=0\(root\)/ };
+    assert_script_run 'toolbox -r touch /etc/passwd', fail_message => 'Root should have access to /etc/passwd!';
+    assert_script_run 'podman ps -a';
+    clean_container_host(runtime => 'podman');
 
+    # Back to root
+    select_console 'root-console';
```
Comment (b10n1k) [inline tests/microos/toolbox.pm:79]: i am not sure if you actually need this. Many modules start with select_console or select_serial_terminal. So do you really need it?

Comment (jlausuch) [reply tests/microos/toolbox.pm:79]: yes, otherwise it will run the next commands in `$user` console...

**Comment by jlausuch** [comment]
Just in case, it also works on SLE Micro GM http://fromm.arch.suse.de/tests/884#

**Pull request merged**